solc/CommandLineInterface.cpp

```diff
@@ -770,14 +849,14 @@ bool CommandLineInterface::processInput()
 		if (successful && m_args.count(g_argFormal))
 			if (!m_compiler->prepareFormalAnalysis())
 				successful = false;
-
-		for (auto const& error: m_compiler->errors())
-			SourceReferenceFormatter::printExceptionInformation(
-				cerr,
-				*error,
-				(error->type() == Error::Type::Warning) ? "Warning" : "Error",
-				scannerFromSourceName
-			);
+		if (!m_compiler->importedSources())
```

Comment:
hmm, I tried that and it means changing a whole bunch of function Calls, for lots of them I did not know exactly whether they could also be called in the context of an imported AST...So I decided to just leave it in.